tests/Integration/WorkloadManagementSystem/Test_ElasticJobDB.py

```diff
@@ -39,9 +40,13 @@ def test_setAndGetJobFromDB(self):
     """
     res = self.jobDB.setJobParameter(100, 'DIRAC', 'dirac@cern')
     self.assertTrue(res['OK'])
+    time.sleep(1)
```

Comment:
why ? if there is a propagation time or something like it, it is good to have it as comment

Reply:
Actually, the index needs to be updated before being queried (as per the test design). Querying index before update would cause errors. Hence, a time delay added to solve the purpose.